src/components/footer/footer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './footer.scss';
+
+function Footer() {
+  return (
+      <footer className="footer">
+        <div className="footer__logo">
+          <img className="footer__logo-image" src="./images/logo.svg" alt="Miami logo" />

```

Comment:
if you have more than two attributes, place each of them on the separate line

Reply:
Fixed!

Reply:
I don't see any changes